### PR TITLE
Add recipe for IgdbApiBundle

### PR DESCRIPTION
**Pull request by EmanuilNikolov**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by EmanuilNikolov** [comment]
Is there something else that I should do, to get merged? I've covered all the requirements in the README.md. If I am missing out on something, please excuse me - this is my first bundle. Will appreciate any feedback. Thank you in advance!

File: emanuilnikolov/igdb-api-bundle/1.0/config/packages/en_igdb_api.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Get your credentials by signing up here: https://api.igdb.com/admin
+en_igdb_api:
+    base_url: "YOUR_BASE_URL"
```
Comment (maxhelias) [inline emanuilnikolov/igdb-api-bundle/1.0/config/packages/en_igdb_api.yaml]: You can use the env var like this :
```suggestion
    base_url: "'%env(YOUR_BASE_URL)%'"
```

File: emanuilnikolov/igdb-api-bundle/1.0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "EN\\IgdbApiBundle\\ENIgdbApiBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "gitignore": [
+        "/config/packages/en_igdb_api.yaml"
+    ]
```
Comment (maxhelias) [inline emanuilnikolov/igdb-api-bundle/1.0/manifest.json]: And add the section env in the manifest like this :
```
   "env": {
        "YOUR_BASE_URL": ""
    }
```

**Comment by EmanuilNikolov** [comment]
Thank you, @maxhelias ! I have applied your suggestions.

**Pull request merged**